src/Commands/StartFrankenPhpCommand.php

```diff
@@ -0,0 +1,215 @@
+
+        $this->forgetEnvironmentVariables();
+
+        $host = $this->option('host');
+
+        $process = tap(new Process([
+            $frankenphpBinary,
+            'run',
+            '-c', $this->configPath(),
+        ], base_path(), [
+            'APP_ENV' => app()->environment(),
+            'APP_BASE_PATH' => base_path(),
+            'APP_PUBLIC_PATH' => public_path(),
+            'LARAVEL_OCTANE' => 1,
+            'SERVER_NAME' => ($this->option('https') ? 'https://' : 'http://')."$host:".$this->getPort(),
+            'WORKER_COUNT' => $this->workerCount() ?: '',
+            'MAX_REQUESTS' => $this->option('max-requests'),
+            'CADDY_SERVER_EXTRA_DIRECTIVES' => $this->getMercureConfig(),
+        ]));
+        $process->setPty($this->input->isInteractive());
+        $server = $process->start();
+
+        $serverStateFile->writeProcessId($server->getPid());
+
+        return $this->runServer($server, $inspector, 'frankenphp');
+    }
+
+    /**
+     * Get the number of workers that should be started.
+     *
+     * @return int
+     */
+    protected function workerCount()
+    {
+        return $this->option('workers') === 'auto'
+            ? 0
+            : $this->option('workers');
+    }
+
+    /**
+     * Get the path to the FrankenPHP configuration file.
+     *
+     * @return string
+     */
+    protected function configPath()
+    {
+        $path = $this->option('frankenphp-config');
+
+        if (! $path) {
+            $basePath = base_path('Caddyfile');
+            touch($basePath);
+
+            return $basePath;
+        }
+
+        if ($path && ! realpath($path)) {
+            throw new InvalidArgumentException('Unable to locate specified configuration file.');
+        }
+
+        return realpath($path);
+    }
+
+    /**
+     * Write the FrankenPHP server state file.
+     *
+     * @return void
+     */
+    protected function writeServerStateFile(
+        ServerStateFile $serverStateFile
+    ) {
+        $serverStateFile->writeState([
+            'appName' => config('app.name', 'Laravel'),
+            'host' => $this->getHost(),
+            'port' => $this->getPort(),
+            'workers' => $this->workerCount(),
+            'maxRequests' => $this->option('max-requests'),
+            'octaneConfig' => config('octane'),
+        ]);
+    }
+
+    /**
+     * Write the server process output to the console.
+     *
+     * @param  \Symfony\Component\Process\Process  $server
+     * @return void
+     */
+    protected function writeServerOutput($server)
+    {
+        [$output, $errorOutput] = $this->getServerOutput($server);
+
+        Str::of($output)
+            ->explode("\n")
+            ->filter()
+            ->each(function ($output) {
+                $this->raw($output);
+            });
+
+        Str::of($errorOutput)
+            ->explode("\n")
+            ->filter()
+            ->each(function ($output) {
+                $this->raw($output);
```

Comment:
This doesn't work properly with --quiet. 
To format it nicely and show requests better, we could do something like this:

```php
        Str::of($errorOutput)
            ->explode("\n")
            ->filter()
            ->each(function ($output) {

                if (Str::contains($output, "http.log.access") && Str::contains($output, "handled request") ) {
                    if ($debug = json_decode(trim(substr($output, strpos($output, "handled request") + 15)), true)) {

                        if (isset($debug['duration'])
                            && isset($debug['request']['method'])
                            && isset($debug['request']['uri'])
                            && isset($debug['status'])) {

                            return $this->requestInfo([
                                'method' => $debug['request']['method'],
                                'url' => $debug['request']['uri'],
                                'statusCode' => $debug['status'],
                                'duration' => $debug['duration'],
                            ]);
                        }

                    }
                }

                $debug = json_decode($output, true);
                if (is_array($debug) && isset($debug['level']) && isset($debug['msg'])) {
                    $extra = Arr::except($debug, ['logger', 'level', 'msg', 'ts']);
                    $msg = (isset($debug['logger']) ? '[' .$debug['logger'] .'] ' : '') .$debug['msg'] . ' ' . ($extra ? json_encode($extra) : '');
                    switch ($debug['level']) {
                        case 'error':
                            return $this->error($msg);
                        case 'warn':
                            return $this->warn($msg);
                        default:
                            return $this->info($msg);
                    }
                }

                $this->raw($output);
            });
```

![image](https://github.com/laravel/octane/assets/973269/0c7f7464-ed78-4ff6-8fb7-3c39f26839b2)